### PR TITLE
fix: replace Non IRGSource G Characters

### PR DESCRIPTION
**Pull request by diauweb**
Replaced some of Non IRG Source G Characters and attached the script did this
Please revert the first commit if it is not ok to add this one-time script

<details>
<summary>Full list</summary>

- bignum.md, (274:28): 诶 is not in kIICore

- queue.md, (115:84): 蒟 is not in IRG G Source

- queue.md, (115:85): 蒻 is not in IRG G Source

- divide-combine.md, (295:46): 増 is not in IRG G Source

- oi.md, (177:10): 報 is not in IRG G Source

- oi.md, (181:6): 選 is not in IRG G Source

- oi.md, (182:6): 選 is not in IRG G Source

- oi.md, (230:9): 資 is not in IRG G Source

- oi.md, (230:10): 訊 is not in IRG G Source

- oi.md, (230:11): 奧 is not in IRG G Source

- oi.md, (230:14): 亞 is not in IRG G Source

- oi.md, (230:15): 競 is not in IRG G Source

- oi.md, (230:16): 賽 is not in IRG G Source

- oi.md, (232:28): 資 is not in IRG G Source

- oi.md, (232:29): 訊 is not in IRG G Source

- oi.md, (236:2): 區 is not in IRG G Source

- oi.md, (236:4): 資 is not in IRG G Source

- oi.md, (236:5): 訊 is not in IRG G Source

- oi.md, (236:6): 學 is not in IRG G Source

- oi.md, (236:10): 競 is not in IRG G Source

- oi.md, (236:11): 賽 is not in IRG G Source

- oi.md, (237:3): 國 is not in IRG G Source

- oi.md, (237:4): 資 is not in IRG G Source

- oi.md, (237:5): 訊 is not in IRG G Source

- oi.md, (237:6): 學 is not in IRG G Source

- oi.md, (237:10): 競 is not in IRG G Source

- oi.md, (237:11): 賽 is not in IRG G Source

- oi.md, (238:2): 資 is not in IRG G Source

- oi.md, (238:3): 訊 is not in IRG G Source

- oi.md, (238:5): 習 is not in IRG G Source

- oi.md, (238:6): 營 is not in IRG G Source

- bitset.md, (95:16): 艹 is not in kIICore

- bitset.md, (97:65): 艹 is not in kIICore

- bfs.md, (237:0): 诶 is not in kIICore

- min-cut.md, (28:39): 増 is not in IRG G Source

- bracket.md, (23:20): 圧 is not in IRG G Source

- lyndon.md, (23:60): 増 is not in IRG G Source

- vim.md, (162:35): 诶 is not in kIICore
</details>

**审核的同学** 请着重关注以下四方面：

1. 注意有没有 typo
2. 不论你是否熟悉相关知识，都请以初学者的角度把这个 PR 的内容阅读一遍，跟着作者的思路走，然后谈谈你的感受
3. 如果你熟悉相关知识，请按照自己的理解评估这个 PR 的内容是否合适
4. 请**尽量**保持跟进直到它被 merge 或 close

### CODE REVIEW

File: docs/ds/divide-combine.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ void build() {
     // 维护单调栈的目的是辅助线段树从 i-1 更新到 i。
     // 更新到 i 后，只需要查询全局最小值即可知道是否有解
 
-    while (tp1 && a[i] <= a[st1[tp1]])  // 单调递増的栈，维护 Min
+    while (tp1 && a[i] <= a[st1[tp1]])  // 单调递增的栈，维护 Min
```
Comment (ksyx) [inline docs/ds/divide-combine.md:296]:
```suggestion
    while (tp1 && a[i] <= a[st1[tp1]])  // 单调递增的栈，维护 Min
```

File: scripts/find_jk.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    read_unihan_db()
+
+    for dirname, _, files in os.walk("../docs"):
+        for f in filter(lambda x: x.endswith(".md"), files):
+            with open(f"{dirname}/{f}", "r") as ff:
+                lns = ff.readlines()
+                process_lines(lns, f)
+    
+if __name__ == "__main__":
+    main()
```
Comment (ksyx) [inline scripts/find_jk.py:58]:
```suggestion
    main()

```

**Review by ksyx — COMMENTED** [review]
thx
顺带 script 包含之前的康熙部首识别吗

**Comment by diauweb** [comment]
> thx
> 顺带 script 包含之前的康熙部首识别吗

这个是不包括的 但是因为康熙部首很好找所以需求不大 如果想考虑自动化的话可以简单加一下

**Review by Xeonacid — APPROVED** [review]
thx

**Review by ksyx — APPROVED** [review]
thx

**Pull request merged**